src/components/contact/FormContact.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Button,
+  Group,
+  SimpleGrid,
+  Text,
+  TextInput,
+  Textarea,
+} from '@mantine/core';
+import { useForm } from '@mantine/form';
+import { zodResolver } from 'mantine-form-zod-resolver';
+import { z } from 'zod';
+
+export const FormContact = () => {
+  const contactSchema = z.object({
+    email: z.string().email("l'email n'est pas valide"),
+    message: z
+      .string()
+      .min(10, 'le message doit contenir au moins 10 caractères'),
+  });
```

Comment:
No validation on the name or subject of mail ? 

Bug -> it's possible to send form, without name and subject : 
![image](https://github.com/FullStackMap/Front/assets/46059868/1cc0a7d5-4fa5-449f-aeeb-a902135f09d1)